internal/bminventory/inventory.go

```diff
@@ -2593,6 +2593,10 @@ func (b *bareMetalInventory) updateOperatorsData(ctx context.Context, cluster *c
 	}
 
 	for _, updatedOperator := range updateOLMOperators {
+		if updatedOperator.Name == "LVM" || updatedOperator.Name == "CNV" {
+			err = errors.Wrapf(err, "failed, either LVM or CNV operator can be installed, both can not be installed")
```

Comment:
```suggestion
			err = errors.Wrapf(err, "Currently, you can not install OpenShift Data Foundation Logical Volume Manager operator at the same time as Virtualization operator.")
```

Reply:
Please use the same text as we will display in the UI, see: https://issues.redhat.com/browse/MGMT-11145

Reply:
This:
Installing OpenShift Data Foundation Logical Volume Manager together with OpenShift Virtualization is not currently allowed.